packages/appstore/src/components/app-launcher/app-launcher.scss

```diff
@@ -13,8 +13,20 @@
     }
 
     &__button {
-        height: fit-content;
         margin-left: auto;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 6px 16px;
+
+        width: 56px;
+        height: 32px;
+
+        /* button/primary-light/default */
+
+        background: rgba(255, 68, 79, 0.16);
```

Comment:
find the color in our constants file.
The best approach is to find the color in our constants file. Then finding its variable in the themes file which is in the same of the constants to prevent facing issues with the dark theme

Reply:
Cannot find it defined anywhere.

Reply:
color-red-6